quickSort.js

```diff
@@ -1,3 +1,4 @@
+// quick sort algorithm
 function quickSort(arr, startIndex, endIndex) {
```

Comment:
Try to see if you can rewrite this so that quickSort only takes in a single parameter, array.